tests/unit/Utils.test.js

```diff
@@ -0,0 +1,73 @@
+import { parseMarkdown } from '../../src/js/utils';
+
+describe('Util ParseMarkdown', () => {
```

Comment:
This is great @philemonputra @jet0612, thanks 🙏. I would also add a test that has multiple MD format text, and the function should also convert the ones that are in MD and should not convert the ones that are not,

Reply:
Got it!

Reply:
Just added a test to accommodate multiple markdown features